docs/pages/integrate.mdx

```diff
@@ -167,7 +167,57 @@ export function FarcasterEmbeds(props: { embeds: Array<Embed> }) {
   return (
     <div>
       {props.embeds.map((embed, i) => (
-        <RenderEmbed embed={embed} key={i} renderers={renderers} />
+        <RenderEmbed 
+          key={i} 
+          renderers={renderers} 
+          
+          // Required context
+          embed={embed} 
+          api={"https://api.modprotocol.org/api"}
```

Comment:
lets add a comment that they can self host the backend in `/examples/api` and pass that url here